src/test/java/ch/wisv/areafiftylan/integration/TicketRestIntegrationTest.java

```diff
@@ -375,7 +373,7 @@ public void testDoTransferAsAnon() {
         //@formatter:on
 
         ttt = tttRepository.findByToken(ttt.getToken()).get();
-        ticket = ticketRepository.findOne(ticket.getId());
+        ticket = ticketRepository.findById(ticket.getId()).orElse(new Ticket());
```

Comment:
I think it is nicer to do `.orElseThrow(TicketNotFoundInDatabaseException::new)` here. That is more explicit and fail-fast.